massa-bootstrap/src/tests/tools.rs

```diff
@@ -238,7 +238,13 @@ pub fn get_random_final_state_bootstrap(
         sorted_ledger.insert(get_random_address(), get_random_ledger_entry());
     }
     // insert the last possible address to prevent the last cursor to move when testing the changes
-    sorted_ledger.insert(Address::from_bytes(&[255; 32]), get_random_ledger_entry());
+
+    let mut bytes = [255; 33];
+    bytes[0] = 1;
```

Comment:
It would be better to have a comment on why we are setting the first byte to a magic value